__tests__/app.test.js

```diff
@@ -303,3 +303,102 @@ describe("/api/articles", () => {
       })
   })
 })
+
+// task 6
+
+describe("/api/articles/:article_id/comment", () => {
+  test("GET: 200 responds with an array of comments for a specific article", (done) => {
+    const { commentData: modifiedCommentData } = testData
+    request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments: receivedComments } = body
+
+        const expectedNumberOfComments = modifiedCommentData.filter(
+          (comment) => {
+            return comment.article_id === 1
+          }
+        ).length
+        expect(receivedComments.length).toBe(expectedNumberOfComments)
+        done()
+      })
+      .catch(done)
+  })
```

Comment:
Not sure why we have a done() here. We also don't need catch blocks in our tests. You can remove these :)